content/blog/deploying-cribl-stream-containers-on-hpe-greenlake.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Deploying Cribl Stream Containers on HPE GreenLake
```

Comment:
Hi! I'm your friendly blog editor. I'll be reviewing your post from a grammatical perspective, as well as for readability and Brand compliance. One thing I like to warn a lot of our writers about up front is that HPE Brand requests that we use "sentence case" for titles, headers, call outs.... everything actually. What that means is that everything except the first word in the sentence, as well as any proper nouns (i.e. names of products, programs, etc.), receive a lower case initial letter. Title case is where you capitalize every "important" word in a sentence. Title case is now frowned upon by HPE Brand.  There is one other instance where you can use initial capital letters and that's when you're phrasing something that the user would see on a screen, i.e. in a screen shot. In other words, a lot of our products were around when Title case was a thing, so their documentation and UI's reflect that.

That being said, here in your title, I personally do not know if "Crible Stream Containers" is the actual name of a product or if "stream" is an adjective and "containers" is a simple noun here. If that is the case, then it should be written out as:
"Deploying Cribl stream containers on HPE GreenLake"

I hope this helps.

Reply:
Cribl Stream is the name of the product. We deploy the Cribl Stream product into Kubernetes containers.

Reply:
Cribl Stream is the name of the software, Cribl is the name of the company.